content/en/dynamic_instrumentation/enabling/java.md

```diff
@@ -45,7 +45,7 @@ Dynamic Instrumentation is a feature of supporting Datadog tracing libraries. If
 
    **Note**: Dynamic Instrumentation is available in the `dd-java-agent.jar` library in versions 1.34.0 and later.
 
-3. Run your service with Dynamic Instrumentation enabled by setting `-Ddd.dynamic.instrumentation.enabled` flag or `DD_DYNAMIC_INSTRUMENTATION_ENABLED` environment variable to `true`. Specify `dd.service`, `dd.env`, and `dd.version` Unified Service Tags so you can filter and group your probes and target active clients across these dimensions.
+3. Run your service with Dynamic Instrumentation enabled by setting `-Ddd.dynamic.instrumentation.enabled` flag or `DD_DYNAMIC_INSTRUMENTATION_ENABLED` environment variable to `true`. Specify `dd.service`, `dd.env`, and `dd.version` Unified Service Tags so you can filter and group your instrumentations and target active clients across these dimensions.
```

Comment:
for dotnet you removed mention of setting service,env,version.
I prefer to highlight setting those unified tags as we must have "service" for our probes to work - as they all bound to a service.

Reply:
I've reverted the change for now to not block this PR. I still think this could be improved, but let's discuss in a follow up PR.